dotcom-rendering/src/components/FootballTableList.tsx

```diff
@@ -0,0 +1,148 @@
+
+const footballTablesGridStyles = css`
+	display: grid;
+	grid-template-columns: [centre-column-start] repeat(4, 1fr) [centre-column-end];
+	column-gap: 10px;
+	${from.mobileLandscape} {
+		column-gap: 20px;
+	}
+
+	${from.tablet} {
+		grid-template-columns: [centre-column-start] repeat(12, 40px) [centre-column-end];
+	}
+
+	${from.desktop} {
+		grid-template-columns: [centre-column-start] repeat(8, 60px) [centre-column-end];
+	}
+
+	${from.leftCol} {
+		grid-template-columns:
+			[left-column-start] repeat(2, 60px)
+			[left-column-end centre-column-start] repeat(8, 60px)
+			[centre-column-end];
+	}
+
+	${from.wide} {
+		grid-template-columns:
+			[left-column-start] repeat(3, 60px)
+			[left-column-end centre-column-start] repeat(8, 60px)
+			[centre-column-end];
+	}
+`;
+
+const CompetitionName = ({ children }: { children: ReactNode }) => (
+	<h2
+		css={css`
+			${textSansBold14}
+			grid-column: centre-column-start / centre-column-end;
+			color: ${palette('--football-competition-text')};
+			border-top: 1px solid ${palette('--football-top-border')};
+			padding: ${space[2]}px;
+			background-color: ${palette('--football-match-list-background')};
+
+			${from.leftCol} {
+				grid-row: 1;
+				display: block;
+				color: ${palette('--football-competition-text')};
+				border-top: 1px solid ${palette('--football-list-border')};
+				background-color: transparent;
+				margin-top: 0;
+				padding: ${space[1]}px 0 0;
+				grid-column: left-column-start / left-column-end;
+				${headlineBold17}
+			}
+		`}
+	>
+		{children}
+	</h2>
+);
+
+const GroupName = ({
+	children,
+	index,
+}: {
+	children: ReactNode;
+	index: number;
+}) => (
+	<h3
+		css={css`
+			grid-column: centre-column-start / centre-column-end;
+			padding-top: ${space[1]}px;
+			padding-bottom: ${space[3]}px;
+			border-top: 1px solid ${palette('--football-list-border')};
+			${headlineBold17}
+
+			${index !== 0 ? `margin-top: ${space[6]}px` : ''}
+		`}
+	>
+		{children}
+	</h3>
+);
+
+type Props = {
+	competitions: FootballTableCompetition[];
+	guardianBaseUrl: string;
+};
+
+export const FootballTableList = ({ competitions, guardianBaseUrl }: Props) => (
+	<Stack space={9}>
+		{competitions.map((competition) => (
+			<section key={competition.name} css={footballTablesGridStyles}>
+				{competition.tables.map((table, groupIndex) => (
+					<>
+						{competition.hasGroups && (
+							<GroupName index={groupIndex}>
+								{table.groupName}
+							</GroupName>
+						)}
+						{
+							// Only show the competition name above/beside the first group
+							groupIndex === 0 && (
+								<CompetitionName>
+									<a
+										href={`${guardianBaseUrl}${competition.url}`}
+										css={css`
+											text-decoration: none;
+											color: inherit;
+											:hover {
+												text-decoration: underline;
+											}
+										`}
+									>
+										{competition.name}
+									</a>
+								</CompetitionName>
```

Comment:
It looks like we've ended up with an inverted structure where the competition `h2`s have become subheadings of the group `h3`s? We might need to take another look at this, perhaps looking again at what frontend is doing to handle this problem?